### DISCUSSION TITLE
data-backdrop is not working on livewire.

### DISCUSSION THREAD

**Discussion by anonymouse703** [twbs/bootstrap#37383] [Q&A] [answered]
Hello! I try to prevent the modal to close when click outside the modal. It's working on my one project but in my new setup it doesn't working.. Does the livewire component has a conflict on it?

```
 <!-- The Modal -->
    <div wire.ignore.self class="modal fade" id="realEstateTypeModal" tabindex="-1" role="dialog" aria-labelledby="realEstateTypeModal" aria-hidden="true" **data-backdrop="static" data-keyboard="false"**>
        <div class="modal-dialog" role="document">
            <livewire:real-estate-type.real-estate-type-form />
        </div>
    </div>
```

**Comment by nkdas91** [comment, marked as answer]
Hi @anonymouse703 

Which version of Bootstrap are you using?

Try replacing
```data-backdrop="static" data-keyboard="false"```

with
```data-bs-backdrop="static" data-bs-keyboard="false"```

post Bootstrap v5, we have to use ```data-bs-*```

**Reply by anonymouse703** [reply]
Thanks sir... I already solved it.
